### PR TITLE
Required Opportunity Fields

### PR DESCRIPTION
**Pull request by mrrigney**
# Critical Changes

# Changes
## New Fields to Support Admissions
- To support the admissions process, we added these fields to the Contact object: SMS Opt Out and Disability Status. We also added these fields to the Opportunity object: Desired Degree Level, Desired Campus, Desired Start Term, Completed FAFSA, and Living Situation. We also updated the HEDA Contact Layout and added a new EDA Opportunity Layout. For more information about configuring these object and working with this fields, see the [EDA Documentation](https://powerofus.force.com/s/article/EDA-Documentation). 

# Issues Closed
[W-9538878](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07AH000000PzaTYAS/view) - R&A
[W-9538879](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07AH000000PzaUYAS/view) - R&A

# New Metadata
### Contact
* Disability Status
* SMS Opt Out
### Opportunity
* Completed FAFSA
* Desired Campus
* Desired Start Term
* Desired Degree Level
* Living Situation
## Page Layout
* EDA Opportunity Layout
* HEDA Contact Layout

## Unpackaged Metadata
* Opportunity page layout assignments

# Testing Notes

### CODE REVIEW

**Comment by mrrigney** [comment]
Running a dev_org flow locally and receiving  [Failed]: Update of Profile Admin: Error on line 74, col 24: In field: layout - no Layout named Contact-EDA Opportunity Layout found
2021-09-24 07:44:01: Exception in task config_dev.deploy_dev_config

Could not process MDAPI response: Update of Profile Admin: Error on line 74, col 24: In field: layout - no Layout named Contact-EDA Opportunity Layout found

**Comment by mrrigney** [comment]
@vmarasow Thinking we should add the new contact fields to the eda contact layout

**Comment by mrrigney** [comment]
@vmarasow Looks like the help text and description fields are not matching some of the spec for example Completed FAFSA

**Comment by mrrigney** [comment]
@vmarasow The page layout for opportunity doesn't seem to match the suggested layout from @Kennen

**Comment by mrrigney** [comment]
@vmarasow Contact Layout should be EDA Contact not HEDA

**Comment by vmarasow** [comment]
> Contact Layout should be EDA Contact not HEDA

I only added the 2 new fields to the existing layout. It was named "HEDA" back when it was added to HEDA, and changing it now would affect existing customers...

**Comment by mrrigney** [comment]
@vmarasow I approve this PR but GitHub won't let me approve as I'm the author of the PR.  Assigning to Bobby and 
Matt.  Nice work!

**Comment by lifewithryan** [comment]
This is on our sprint board to review next week. cc: @MatthewBlanski

File: force-app/main/default/objects/Opportunity/fields/Desired_Degree_Level__c.field-meta.xml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    <trackTrending>false</trackTrending>
+    <type>Picklist</type>
+    <valueSet>
+        <restricted>true</restricted>
+        <valueSetDefinition>
+            <sorted>false</sorted>
+            <value>
+                <fullName>Non-degree</fullName>
+                <default>false</default>
+                <label>Non-degree</label>
```
Comment (agoolsby) [inline force-app/main/default/objects/Opportunity/fields/Desired_Degree_Level__c.field-meta.xml:19]:
```suggestion
                <label>Non-Degree</label>
```

File: force-app/main/default/objects/Opportunity/fields/Completed_FAFSA__c.field-meta.xml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomField xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>Completed_FAFSA__c</fullName>
+    <defaultValue>false</defaultValue>
+    <description>Indicates whether a FAFSA application related to this Opportunity has been completed.</description>
```
Comment (agoolsby) [inline force-app/main/default/objects/Opportunity/fields/Completed_FAFSA__c.field-meta.xml:5]:
```suggestion
    <description>Indicates whether a FAFSA application related to this Opportunity was completed.</description>
```

File: force-app/main/default/objects/Opportunity/fields/Completed_FAFSA__c.field-meta.xml [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+    <externalId>false</externalId>
+    <inlineHelpText>Indicates whether a FAFSA application related to this Opportunity has been completed.</inlineHelpText>
```
Comment (agoolsby) [inline force-app/main/default/objects/Opportunity/fields/Completed_FAFSA__c.field-meta.xml:7]:
```suggestion
    <inlineHelpText>Indicates whether a FAFSA application related to this Opportunity was completed.</inlineHelpText>
```

**Review by agoolsby — CHANGES_REQUESTED** [review]
@vmarasow minor changes for you.

**Review by agoolsby — APPROVED** [review]

**Comment by robertbwatson** [comment]
@vmarasow @mrrigney @MatthewBlanski Hi -- I'm going to be on PTO starting on Tuesday. If this is a next sprint item to review, is this something that could be reviewed by Jim and Matt? I'm not sure it needs all 3 of us to approve.

**Comment by mrrigney** [comment]
> @vmarasow @mrrigney @MatthewBlanski Hi -- I'm going to be on PTO starting on Tuesday. If this is a next sprint item to review, is this something that could be reviewed by Jim and Matt? I'm not sure it needs all 3 of us to approve.

@robertbwatson That is fine, someone mentioned your name as a reviewer.  I have reviewed so thinking it is something @MatthewBlanski to look at.  There was some talk of putting it this sprint which would mean reviewed today but can also wait until next sprint

**Review by MatthewBlanski — APPROVED** [review]

**Pull request merged**